src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -39,25 +39,24 @@ Factors affecting access to the migration wizard:
 
-## Think about what groups you'll need [#plan-groups]
+To learn more, see [User management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts).
 
-Once you have a decent understanding of the user management concepts, you'll want to think about how you'll implement groups for your organization. The more complex your organization is, the more it may help you to first plan out how your user groups and access grants will be structured. 
 
-Here are some tips for how different types of organizations may use groups:  
+## Think about the groups and roles you'll need [#plan-groups]
 
-* For organizations with multiple accounts, a common configuration would include a group with the **Organization manager** role, a group with the **Authentication domain manager**, a group with the **Billing user** at the master account level, and then configuring product admins, users, and all other roles at the sub-account level, depending on organization preferences.  
-* Customers with smaller or flatter organizations that require transparency for all teams across all of their data could have as little as 1 or 2 groups.
-* For organizations where all data is in a single account, a common configuration might be around five or six groups.
+If you'll need to do a good amount of partitioning of users' access to accounts or roles, you'll want to think about how you'll implement groups for your organization. Here are some tips for how different types of organizations may use groups:  
 
-As an optional step, we've created [a planning spreadsheet](https://docs.google.com/spreadsheets/d/1-0saEkEFXnLjwjLXhoSAEAP9lbU0iaZzjMEraGNJOME/edit?usp=sharing) to help you plan out the groups and access grants you'll need. This is only for planning purposes as a reference guide: it's not intended to be uploaded. Please make a copy and follow the instructions. At the appropriate point in the user migration wizard workflow, you can use this planning resource to help you create user groups and access grants. 
+* For organizations with multiple accounts, a common configuration would include a group with the [**Organization manager** role](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#roles), a group with the [**Authentication domain manager**](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#roles), a group with the [**Billing user**](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#roles) at the master account level, and then configuring product admins, users, and all other roles at the sub-account level, depending on organization preferences.  
```

Comment:
Maybe considering break this into two sentences starting at "and then configuring product admins..."